simvue/config/parameters.py

```diff
@@ -21,25 +21,29 @@
 
 
 class ServerSpecifications(pydantic.BaseModel):
-    url: pydantic.AnyHttpUrl
-    token: pydantic.SecretStr
+    url: pydantic.AnyHttpUrl | None
+    token: pydantic.SecretStr | None
 
     @pydantic.field_validator("url")
     @classmethod
     def url_to_api_url(cls, v: typing.Any) -> str:
-        if f"{v}".endswith("/api"):
-            return f"{v}"
-        _url = URL(f"{v}") / "api"
-        return f"{_url}"
+        if v:
```

Comment:
Can we swap these to be `if not v: return` followed by alternative case block instead?
Also typing should be set to `str | None` as you are now allowing no value at all